rancid-tomatillos/src/Components/Header/Header.js

```diff
@@ -1,27 +1,37 @@
 import React from 'react';
 import './Header.scss';
 import { connect } from 'react-redux';
-import { Link } from 'react-router-dom';
+import { Route, Link } from 'react-router-dom';
 import { removeUser, updateLoggedIn } from '../../actions/index';
 
 
-const Header = ({ data, isLoggedIn, removeUser, updateLoggedIn }) => {
+const Header = ({ data, isLoggedIn, removeUser, updateLoggedIn, path }) => {
   let log;
```

Comment:
Like this functionality, and wonder if it would be possible to apply it so that they are redirected back to the page they were just on after they have logged in. Ex: User visits page, goes to MoviePage before logging in, logs in and is redirected back to the MoviePage rather than the home page and having to click through again. Not the biggest of deals on a project this small but just thinking if there was more levels to this site, being redirected to the beginning could be a drag.

Reply:
I also agree that it would probably make more sense for UI if they are redirected back to whatever page they were on before logging in. Though, we made it this way because it specifically says to have them redirected to the home page after login in the comp. 'After logging in, the user knows they are logged in and they are taken back to the homepage.'